code/src/components/TaskList/TaskList.css

```diff
@@ -0,0 +1,56 @@
+.empty-list-section {
+  color: #9e9e9e;
+  padding: 5em;
+}
+
+.emoji-span {
+  margin-left: 3.5em;
+}
+
+.list-section {
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.task-container {
+  background-color: #1b1b24;
+  border: 2px solid black;
+  border-radius: 12px;
+  -webkit-border-radius: 12px;
+  -moz-border-radius: 12px;
+  -ms-border-radius: 12px;
+  -o-border-radius: 12px;
+  box-shadow: 6px 6px 1px 1px #000;
+  display: inline-flex;
+  padding: 0.1em 0;
+  position: relative;
+  margin-top: 3%;
+  margin-left: 5%;
+  margin-right: auto;
+  width: 90%;
+}
+
+.task-text {
+  flex-grow: 1;
+  max-width: 75%;
+  padding: 0.3em 0.2em;
+  position: relative;
+}
+
+.delete-button {
```

Comment:
The "X" looks great! However, it could be clearer that this is in fact a button, perhaps by using the same background as for the task button, and by adding "cursor: pointer".

Reply:
Indeed, thanks for the tip! Implemented :)